src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2314,6 +2314,80 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/search/{id}/metadata:
```

Comment:
One thing I want to make sure of is that we continue to limit that this is seen in prod (since this is a feature flagged...feature). Is this going to expose the search api?  We should add a filter in SwaggerDocumentationConfig is so

Reply:
This is feature flagged behind the search api just like the other search functionality.

Reply:
Cool, I wasn't sure since it also had the snapshot tag